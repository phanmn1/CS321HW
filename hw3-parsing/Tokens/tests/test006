1MoreExample
   (0, a10TimesMoreInterestingIdentifier, 39)
