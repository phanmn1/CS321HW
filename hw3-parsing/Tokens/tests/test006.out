Invalid syntax at (0,0), null
